lasagne/layers/conv.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, incoming, num_filters, filter_length, stride=1,
             self.nonlinearity = nonlinearity
 
         self.num_filters = num_filters
-        self.filter_length = filter_length
+        self.filter_size = filter_size
```

Comment:
That's a very good point, although it makes it slightly more awkward when specifying it as an integer. `pool_shape=3` looks a bit weird to me because I've been conditioned to expect a shape to look like a tuple, and I think this goes for most numpy / Theano users :)

But of course there is a lot to be said for consistency. Would love to hear @f0k's thoughts on this (and well, anyone else's)!